assets/scss/main.scss

```diff
@@ -68,6 +68,37 @@ body {
   margin: 0 auto;
 }
 
+/* Halloweeeeeeen 2019  */
+:root {
+  --width: 20px;
+  --x-speed: 1.566456s;
+  --y-speed: 2.7341s;
```

Comment:
spooky boi is flying too fast around my screen can you make spooky boi fly slower :(